src/construct.ts

```diff
@@ -217,13 +217,13 @@ export class Node {
    * Context is usually initialized at the root, but can be overridden at any point in the tree.
    *
    * @param key The context key
-   * @returns The context value or `undefined` if there is no context value for thie key.
+   * @returns The context value or `undefined` if there is no context value for this key.
    */
-  public tryGetContext(key: string): any {
+  public tryGetContext<ContextValue = any>(key: string): ContextValue | undefined {
```

Comment:
@fargito Unfortunately I don't think this would work since [jsii] doesn't support generics.

[jsii]: https://github.com/aws/jsii


Perhaps @rix0rrr or @RomainMuller would have some ideas.

Reply:
This is unfortunate, I guess that if they don't have an idea, I'll close the issue